### ISSUE TITLE
Does not work with Intellij 12.0

### ISSUE THREAD

**Issue by MarcinKuder** [codebling/CommunityCase#1]

**Comment by codebling** [comment]
Hey, thanks for trying it out! Can you be a bit more specific with regards to what is not working? Does the plugin throw exceptions when IntelliJ starts up? Does it load alright but not work? What version of ClearCase are you using?

**Comment by MarcinKuder** [comment]
Yes, sorry I was not specific.
Loading the original (dowloaded from github) CommunityCase.jar file gives an error that the plugin is not compatible with current IntelliJ version 12.1 (IC-129.60).
I was able to compile it from sources using version 11 (IC-117.963) of the Idea SDK but trying to import the plugin jar into 12.1 gives the same error. 
Compiling using Idea SDK 12.1 (IC-129.17) gives lots of errors - e.g.
com.intellij.openapi.vcs.annotate.AnnotationListener not found, 
com.intellij.openapi.vcs.changes.LocalChangeList has some new methods that needs to be implemented,
etc..
I looks like the API had some significant changes between 11 and 12.

**Comment by codebling** [comment]
Ah ok. They tend to change the API a lot. I think that the last version I used the plugin with was 11. Let me have a look, it shouldn't be too hard to fix.

**Comment by codebling** [comment]
In other news, you can download the binaries without having to compile them yourself. From the. jar at the root of every branch (even though binaries really don't belong in a Git repo -- for a while I was uploading the jars to GitHub but I think I stopped because it was getting too tedious)

**Comment by codebling** [comment]
I just got back a few hours ago from several months of travelling, can you give me a few days to look at this? Alternatively, if you want to try to make the changes yourself, CommunityCase is based on the IntelliJ git4idea plugin and the source is available online. Whenever I get lost in the API changes, I usually look at the problematic corresponding files in the git4idea source to see what has changed.

**Comment by MarcinKuder** [comment]
Thanks. No need to rush. I will try to look into the git4idea sources as
you pointed out and see if I can fix it.

**Comment by codebling** [comment]
Hey, I've made all of the trivial changes, just looking at the new diffmerge change that was introduced, trying to figure out how to port it

**Comment by codebling** [comment]
Had to abandon the diffmerge change, it's been too long since I've worked with ClearCase to remember how to do a few things (almost 2 years) and I have no environment to test in. Anyways, I've pushed a new 12.0.4 branch which contains a prebuilt plugin jar that you can try out. Let me know if you have any problems. Have you been using this successfully with 11? I have received feedback only once on this project, it was negative and the user refused to even describe the problem to me, saying only that I had designed the plugin poorly (?).

**Comment by MarcinKuder** [comment]
I have not used it before. I just found it recently and wanted to give it a
try. Currently I am using my own clearcase plugin but it is not a fully
version control plugin but rather a simple menu items to run checkout and
version tree commands for currently selected file. I wanted something
better and this is how I found your project.
Thanks a lot for your time, I will checkout the 12.0.4 branch, see if it
works and report back.

Regards,
Marcin
